tasks/amplify.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Check if Amplify Agent is installed
+  package:
+    name: nginx-amplify-agent
+    state: present
+  ignore_errors: true
+  register: amplify_agent_installed
```

Comment:
Is this compatible with all distribution compatible with this role ? (like Debian, Freebsd, Redhat, ...)
If not can you put a `when` condition ? (And we should indicated support OS in the documentation)

Reply:
Also, it's a packages installation, should be tagged as "packages" and not "configuration"

Reply:
According to their website (https://www.nginx.com/products/nginx-amplify/#tech-specs), the list of supported OS's: 

- Amazon Linux
- CentOS
- RHEL
- Debian
- Ubuntu

Not sure about OpenSUSE, FreeBSD and SmartOS unfortunately. I'll add the check accordingly.